packages/block-library/src/image/block.json

```diff
@@ -80,6 +80,10 @@
 			"source": "attribute",
 			"selector": "figure > a",
 			"attribute": "target"
+		},
+		"enableLightbox": {
```

Comment:
Do you know if using the `enableX` is used somewhere else to define an attribute name? From what I can see attributes usually don't include that syntax. Should we just use `lightbox`?

Reply:
In https://github.com/WordPress/gutenberg/pull/49972 it's a `lightbox` field inside the `behaviors` object. Let's align both implementations.